app/src/main/java/ch/epfl/sweng/eventmanager/repository/ConcertRepository.java

```diff
@@ -0,0 +1,53 @@
+package ch.epfl.sweng.eventmanager.repository;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.support.annotation.NonNull;
+import android.util.Log;
+import ch.epfl.sweng.eventmanager.repository.data.Concert;
+import ch.epfl.sweng.eventmanager.repository.data.Event;
+import com.google.firebase.database.*;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Collections;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * @author Louis Vialar
+ */
+@Singleton
+public class ConcertRepository {
+    private static String TAG = "ConcertRepository";
```

Comment:
Should it not be in the strings.xml ? Otherwise, I think it should be final no ?

Reply:
Should be final yup, I let you change that